### ISSUE TITLE
get all groups

### ISSUE THREAD

**Issue by kilchenmann** [dhlab-basel/Knora#587]
I'm trying to get a list of existing groups in Knora.
With the route http://localhost:3333/v1/groups I only get one group. But there should be more than the "Image reviewer" group? Or I'm wrong?

![screen shot 2017-07-28 at 11 38 50](https://user-images.githubusercontent.com/4253580/28712048-e8307176-7389-11e7-98de-14b2705ff899.png)

**Comment by subotic** [comment]
Currently, you only get the custom groups back and not the built-in groups.

I suppose that the user would expect to see all groups here.

**Comment by subotic** [comment]
so, you would like to see all (custom and built-in) groups here?

**Comment by kilchenmann** [comment]
Yes, it would be nice to have one or (perhaps better) two lists of the groups: custom and built-in. 
Otherwise — we already discussed it — I have to know the values (group IRI) of the built-in groups; but I don't like this kind of hard coded stuff.

**Comment by subotic** [comment]
ok, I can return all groups and also add an additional property `type` which can have `user` or `system` as value.

also, to make it less confusing for the user, to get detailed information about a group, the request will always need to include the IRI of the group and the project, e.g.,

```
GET: http://localhost:3333/v1/groups/<groupIri>/<projectIri>
```

This way, anything returned by `v1/groups` can be used the same way. Now I could even return the members of each group.
